message_ix_models/data/node/ZMB.yaml

```diff
@@ -0,0 +1,9 @@
+# Codes for the "node" dimension of the MESSAGE-IL model
+
+World:
+  name: World
+  description: MESSAGE-ZA regions
+
+Zambia:
+  name: Zambia
+  parent: World
```

Comment:
We found with MESSAGEix-Israel that it was important to have the `child:` field here with the ISO 3166-1 alpha-3 code for Israel:

https://github.com/iiasa/message-ix-models/blob/be551575085beb0d223c27e0aac5bbd392a6babf/message_ix_models/data/node/ISR.yaml#L7-L10

Can you please add that?

Among other things, this will allow to use this list with other code that selects/aggregates data from data sources that are at country resolution.